Content.Client/_DV/UserInterfaces/Systems/Cwoink/CHelpUIController.cs

```diff
@@ -0,0 +1,555 @@
+        UIHelper?.Open(userId, _discordRelayActive);
+    }
+
+    public void ToggleWindow()
+    {
+        EnsureUIHelper();
+        UIHelper?.ToggleWindow();
+    }
+
+    public void PopOut()
+    {
+        EnsureUIHelper();
+        if (UIHelper is not CuratorCHelpUIHandler helper)
+            return;
+
+        if (helper.Window == null || helper.Control == null)
+        {
+            return;
+        }
+
+        helper.Control.Orphan();
+        helper.Window.Dispose();
+        helper.Window = null;
+        helper.EverOpened = false;
+
+        var monitor = _clyde.EnumerateMonitors().First();
+
+        helper.ClydeWindow = _clyde.CreateWindow(new WindowCreateParameters
+        {
+            Maximized = false,
+            Title = "Curator Help",
+            Monitor = monitor,
+            Width = 900,
+            Height = 500
+        });
+
+        helper.ClydeWindow.RequestClosed += helper.OnRequestClosed;
+        helper.ClydeWindow.DisposeOnClose = true;
+
+        helper.WindowRoot = _uiManager.CreateWindowRoot(helper.ClydeWindow);
+        helper.WindowRoot.AddChild(helper.Control);
+
+        helper.Control.PopOut.Disabled = true;
+        helper.Control.PopOut.Visible = false;
+    }
+
+    private void UnreadCHelpReceived()
+    {
+        GameCHelpButton?.StyleClasses.Add(MenuButton.StyleClassRedTopButton);
+        _hasUnreadCHelp = true;
+    }
+
+    private void UnreadCHelpRead()
+    {
+        GameCHelpButton?.StyleClasses.Remove(MenuButton.StyleClassRedTopButton);
+        _hasUnreadCHelp = false;
+    }
+
+    public void OnStateEntered(GameplayState state)
+    {
+        if (GameCHelpButton != null)
+        {
+            GameCHelpButton.OnPressed -= CHelpButtonPressed;
+            GameCHelpButton.OnPressed += CHelpButtonPressed;
+            GameCHelpButton.Pressed = UIHelper?.IsOpen ?? false;
+
+            if (_hasUnreadCHelp)
+            {
+                UnreadCHelpReceived();
+            }
+            else
+            {
+                UnreadCHelpRead();
+            }
+        }
+    }
+
+    public void OnStateExited(GameplayState state)
+    {
+        if (GameCHelpButton != null)
+            GameCHelpButton.OnPressed -= CHelpButtonPressed;
+    }
+
+    public void OnStateEntered(LobbyState state)
+    {
+    }
+
+    public void OnStateExited(LobbyState state)
+    {
+    }
+}
+
+// please kill all this indirection
+public interface ICHelpUIHandler : IDisposable
+{
+    public bool IsCurator { get; }
+    public bool IsOpen { get; }
+    public void Receive(CwoinkTextMessage message);
+    public void Close();
+    public void Open(NetUserId netUserId, bool relayActive);
+    public void ToggleWindow();
+    public void DiscordRelayChanged(bool active);
+    public void PeopleTypingUpdated(CwoinkPlayerTypingUpdated args);
+    public event Action OnClose;
+    public event Action OnOpen;
+    public Action<NetUserId, string, bool, bool>? SendMessageAction { get; set; }
```

Comment:
Every member of an interface is public
```suggestion
    bool IsCurator { get; }
    bool IsOpen { get; }
    void Receive(CwoinkTextMessage message);
    void Close();
    void Open(NetUserId netUserId, bool relayActive);
    void ToggleWindow();
    void DiscordRelayChanged(bool active);
    void PeopleTypingUpdated(CwoinkPlayerTypingUpdated args);
    event Action OnClose;
    event Action OnOpen;
    Action<NetUserId, string, bool, bool>? SendMessageAction { get; set; }
```